### PR TITLE
Add improvements to hint chooser UI.

### PR DESCRIPTION
**Pull request by dlongley**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
 
+### Fixed
+- Add `WrmOriginCard` component.
+- Add `message` slot for hint chooser.
+- Simplify hint chooser UI (remove cancel
+  button when confirm button is not shown).
+- Provide method for getting web app manifest
+  icon.
+- Improve CSS for close buttons and more
+  responsive modals.
```
Comment (davidlehn) [inline CHANGELOG.md]: I find it odd these changes and the rest of the file are word wrapped at like 50 chars.

File: WrmHint.vue [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
-      style="width: 40px; max-height: 40px" class="wrm-flex-item">
+      style="width: 32px; max-height: 32px" class="wrm-flex-item">
     <div style="margin-left: 10px;" class="wrm-flex-item-grow">
       <strong>{{hint.name}}</strong>
-      <h6>{{hint.origin}}</h6>
+      <div class="wrm-flex-row" style="justify-content: flex-start">
+        <i class="fas fa-lock wrm-flex-item wrm-green"
+          style="padding-right: 2px"></i>
+        <h6>
+          <span class="wrm-green">https</span>://{{hint.origin.substr(8)}}</span>
```
[the remark this answers is not in the document]
Comment (davidlehn) [reply WrmHint.vue]: Probably missing wrm-dark-gray style like elsewhere?

[the remark this answers is not in the document]
Comment (dlongley) [reply WrmHint.vue]: I'm not going to address this now ... fixed in a later branch.

File: index.js [hunk 3]
```diff
@@ -8,10 +8,15 @@
...
 
 export function install(Vue, options) {
   Vue.component(
     'wrm-hint-chooser', () => import('./WrmHintChooser.vue'));
+  Vue.component(
+    'wrm-origin-card', () => import('./WrmOriginCard.vue'));
   Vue.component(
     'wrm-permission-dialog', () => import('./WrmPermissionDialog.vue'));
+  Vue.component(
+    'wrm-header-close-button', () => import('./WrmHeaderCloseButton.vue'));
```
Comment (davidlehn) [inline index.js]: Could put this in alpha order.

**Pull request merged**